src/main/java/net/minecraftforge/event/entity/living/LivingSpawnEvent.java

```diff
@@ -98,6 +122,7 @@ public CheckSpawn(EntityLiving entity, World world, float x, float y, float z)
         }
 
         public boolean isSpawner() { return isSpawner; }
+        public @Nullable MobSpawnerBaseLogic getSpawner() { return spawner; }
```

Comment:
`@Nullable` should go on the line before, except in patches where size is being minimized.